Modules/Core/Sources/Core/MediaType/MediaType.swift

```diff
@@ -1,7 +1,7 @@
-public protocol MediaTypeRepresentor {
+public protocol MediaTypeConvertible {
```

Comment:
Makes more sense now that I read more closely. `Convertible` is a little confusing but fine overall.